sdks/python/apache_beam/runners/dataflow/internal/names.py

```diff
@@ -36,10 +36,10 @@
 
 # Update this version to the next version whenever there is a change that will
 # require changes to legacy Dataflow worker execution environment.
-BEAM_CONTAINER_VERSION = 'beam-master-20220512'
+BEAM_CONTAINER_VERSION = 'beam-master-20220526'
 # Update this version to the next version whenever there is a change that
 # requires changes to SDK harness container or SDK harness launcher.
-BEAM_FNAPI_CONTAINER_VERSION = 'beam-master-20220512'
+BEAM_FNAPI_CONTAINER_VERSION = 'beam-master-20220526'
```

Comment:
I pushed a commit to bump the Dataflow container, since our internal worker code also had a few places where we re-entered `ScopedState` instances.